### PR TITLE
refactor: updates from UI review

### PR DESCRIPTION
**Pull request by satello**
- Allow users to input their own address as the other party
- Hide Invoices tab
- Update template
- Remove Custom template
- remove "Remaining" in the instructions text

### CODE REVIEW

File: src/bootstrap/app.js [hunk 1]
```diff
@@ -44,17 +44,6 @@ const Main = ({ children }) => (
           />
         </li>
         <li className="menu-invoice" onClick={() => navigate('/new/invoice')}>
```
Comment (epiqueras) [inline src/bootstrap/app.js]: Why not remove the full `li`.

File: src/constants/templates.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 const templates = [
   {
-    value: 'general',
-    label: 'General',
+    value: 'cryptocurency-transaction',
```
Comment (epiqueras) [inline src/constants/templates.js]: Typo.

File: src/constants/templates.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
-  },
-  { value: 'custom', label: 'Custom', content: '' }
+      'The purpose of this contract is to provide clear guidelines for [Seller] and [Buyer] of [cryptoasset] in order to facilitate the transfer of said asset. \n\n[Buyer] deposits funds into the escrow.\n\n[Seller] will send [cryptoasset] into [address] within [due date].\n\nOnce [Buyer] receives [cryptoasset] they release funds to seller.\n\nIf [seller] does not release [cryptoasset] before contract end date and no dispute is brought, the contract automatically sends [cryptoasset] to [buyer]'
```
Comment (epiqueras) [inline src/constants/templates.js]: This is outdated. See https://docs.google.com/document/d/1LzDgLpXIU81byNi2A2bBb1_kL9Km7jVFrAPMcUPWObA/edit#heading=h.77zcrpw9i7y3

Comment (satello) [reply src/constants/templates.js]: So there is no contract text at all then. We should remove that whole text box

Comment (epiqueras) [reply src/constants/templates.js]: It becomes the meta evidence `description`. We need it there.

**Pull request merged**